nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp

```diff
@@ -300,6 +302,83 @@ bool BtActionServer<ActionT, NodeT>::loadBehaviorTree(const std::string & bt_xml
   return true;
 }
 
+template<class ActionT, class NodeT>
+bool BtActionServer<ActionT, NodeT>::loadBehaviorTreesRecursive(
```

Comment:
Should we add the recursive check to `loadBehaviorTree` instead? I think this feature can be additive without needing a whole new function (creating some dead code with `loadBehaviorTree`)